handlers/zuora-callout-apis/src/main/scala/com/gu/autoCancel/AutoCancel.scala

```diff
@@ -36,17 +35,20 @@ object AutoCancel extends Logging {
    * invoices should be 0.
    */
   private def executeCancel(requests: Requests, dryRun: Boolean)(acRequest: AutoCancelRequest): ApiGatewayOp[Unit] = {
-    val AutoCancelRequest(accountId, subToCancel, cancellationDate, invoiceId, invoiceAmount) = acRequest
+    val AutoCancelRequest(accountId, subToCancel, cancellationDate, invoiceId) = acRequest
```

Comment:
I'm getting confused with all the invoices
```suggestion
    val AutoCancelRequest(accountId, subToCancel, cancellationDate, overdueInvoiceId) = acRequest
```